src/test/java/ua/kpi/iasa/IASA_Organiser/controller/ControllerTest.java

```diff
@@ -97,7 +97,7 @@ public void shouldCreateNewEvent() {
     public void shouldChangeEvent() {
         controller.setEventService(eventService);
 
-        controller.changeEvent(event);
+        controller.updateEvent(event);
```

Comment:
Please make git fetch

Reply:
Fix it and call me in Telegram group when you done that task

Reply:
Made it)